common/app/conf/switches/PerformanceSwitches.scala

```diff
@@ -5,24 +5,15 @@ import org.joda.time.LocalDate
 
 trait PerformanceSwitches {
 
-  val MinifyInlineJsSwitch = Switch(
+  val ClosureCompilerStandardOptimisation = Switch(
```

Comment:
maybe `InlineJSStandardOptimisation`? It's not closure that's the thing here, it's which JS is compressed